### ISSUE TITLE
Crash on displaying sound message

### ISSUE THREAD

**Issue by bepaald** [signalapp/Signal-Android#9676]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Signal crashes when an audio message comes into view. This seems related to #9674, but the crash is different. That one has a division by zero, whereas I see:
```
05-31 01:06:19.513  8790  8839 E AndroidRuntime: FATAL EXCEPTION: signal-bounded-3
05-31 01:06:19.513  8790  8839 E AndroidRuntime: Process: org.thoughtcrime.securesms, PID: 8790
05-31 01:06:19.513  8790  8839 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'long java.lang.Long.longValue()' on a null object reference
05-31 01:06:19.513  8790  8839 E AndroidRuntime: 	at android.media.MediaFormat.getLong(MediaFormat.java:612)
05-31 01:06:19.513  8790  8839 E AndroidRuntime: 	at org.thoughtcrime.securesms.audio.AudioWaveForm.generateWaveForm(AudioWaveForm.java:94)
05-31 01:06:19.513  8790  8839 E AndroidRuntime: 	at org.thoughtcrime.securesms.audio.AudioWaveForm.lambda$generateWaveForm$3$AudioWaveForm(AudioWaveForm.java:66)
05-31 01:06:19.513  8790  8839 E AndroidRuntime: 	at org.thoughtcrime.securesms.audio.-$$Lambda$AudioWaveForm$5d0rAHsZCuSR9OItWuEmVwJqr3M.run(lambda)
05-31 01:06:19.513  8790  8839 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
05-31 01:06:19.513  8790  8839 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
05-31 01:06:19.513  8790  8839 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:818)
```

### Steps to reproduce
It seems that any audio message coming into view crashes the app. I scrolled back to an old audio message and Signal crashed. Then, thinking it might be a problem with that particular message, I went to note-to-self and recorded an audio message to myself. As soon as I let go of the record button (when the newly recorded audio message is given a message bubble), the app crashes. After restarting, the app crashes as soon as I try to enter the note-to-self thread.

### Device info
**Device:** ZTE Z981
**Android version:** 6.0.1
**Signal version:** 4.61.0

### Link to debug log
https://debuglogs.org/3a2e0945e6f3c7cc11a0ea79958996c5bbdcfdf0002b3e9fbd3780dba901f38a

On my other beta phone (a Sony Xperia Z5C, with android 7.1.1), everything seems to work fine by the way.

**Comment by Sgn-32** [comment]
as Workaround try to delete the voice message in note to self in this way:
- open Signal settings
- Storage
- Review Storage
- Audio
- Search for the voice message in note to self
- long press on this message
- in top bar, you can select the trash

**Comment by bepaald** [comment]
@Sgn-32: thanks, that would actually have been a clever solution. It was just on one of my beta-test-phones though, so I just deleted the entire note-to-self thread and nothing important was lost.

Anyway, this issue appears to have been fixed in 4.61.3 (no audio previews, but no crashes either), so I'm closing this. Thanks!
